charts/gp-reporting-kafka-health/v0.0.1/questions.yml

```diff
@@ -0,0 +1,140 @@
+    options:
+      - prime
+      - qa
+      - testing
+      - 01
+      - 02
+      - 03
+      - 04
+      - 05
+      - 06
+      - 07
+      - 08
+      - 09
+      - 10
+    default: "01"
+
+  ####
+  #### Reporting Kafka Health Service Settings
+  ####
+  - variable: build.commitSHA
+    type: string
+    required: true
+    group: "Application"
+    label: Application Build SHA
+    description: The commit SHA of the GitPrime Application build you want to deploy.
+    default: latest
+
+  - variable: datadog.enabled
+    type: boolean
+    required: true
+    group: "Application"
+    label: "Enable DataDog"
+    description: "Turn DataDog APM on for the pipeline"
+
+  - variable: dns.ndots
+    type: string
+    required: false
+    group: "Application"
+    label: "Number of ndots for DNS configuration"
+    description: "Modifies the number of ndots in /etc/resolv.conf"
+    default: 2
+
+  # Reporting Kafka Health Service Settings
+  - variable: application.webPort
+    type: string
+    required: true
+    group: "Application / Reporting Kafka Health Service"
+    label: Application port binding
+    description: The port binding that the application uses to listen to web requests
+    default: "8080"
+
+  - variable: application.nodePort
+    type: string
+    required: true
+    group: "Application / Reporting Kafka Health Service"
+    label: Application node port
+    description: The port that is exposed on the node which routes to the app port binding
+    default: "30780"
+
+  - variable: application.replicaCount
+    type: string
+    required: true
+    group: "Application / Reporting Kafka Health Service"
+    label: Number of Pods for services
+    description: The number of pods to deploy for the services that handle new repositories.
+    default: 1
+
+  - variable: application.javaOptions
+    type: string
+    required: true
+    group: "Application / Reporting Kafka Health Service"
+    label: Reporting Kafka Health Service Java Options
+    description: The Java options for the services.
+    default: "-d64 -Xms2g -Xmx4g"
+
+  - variable: application.resources.requests.cpu
+    type: string
+    required: true
+    group: "Application / Reporting Kafka Health Service"
+    label: Reporting Kafka Health Service Requested CPU
+    description: The number of CPUs the services will request.
+    default: "100m"
+
+  - variable: application.resources.requests.memory
+    type: string
+    required: true
+    group: "Application / Reporting Kafka Health Service"
+    label: Reporting Kafka Health Service Requested Memory
+    description: The maximum amount of memory a worker will request.
+    default: "4Gi"
+
+  - variable: application.resources.limits.cpu
+    type: string
+    required: true
+    group: "Application / Reporting Kafka Health Service"
+    label: Reporting Kafka Health Service CPU Limit
+    description: The maximum number of CPUs the services can use.
+    default: "4"
```

Comment:
This should probably be lower. 1 Should be fine.

Reply:
Done